src/openforms/authentication/contrib/digid_eherkenning_oidc/views.py

```diff
@@ -61,14 +62,28 @@ def get(self, request):
 
 
 class OIDCAuthenticationCallbackView(_OIDCAuthenticationCallbackView):
+    def get_error_message_parameters(
+        self, parameter: str, problem_code: str
+    ) -> tuple[str, str]:
+        """
+        Return a tuple of the parameter type and the problem code.
+        """
+
+        return (parameter, problem_code)
```

Comment:
```suggestion
        return (parameter or BACKEND_OUTAGE_RESPONSE_PARAMETER, problem_code or self.plugin_identifier)
```
probably should have this as fallback behaviour in case the error/description are empty?